charts/factorio-server-charts/Chart.yaml

```diff
@@ -20,7 +20,7 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.12
+version: 1.0.13
```

Comment:
can you bump this to .14 :) I already increased this on the main branch

Reply:
Yeah sure, commit incoming